packages/sdk/app-graph/src/graph-builder.test.ts

```diff
@@ -240,34 +275,31 @@ describe('GraphBuilder', () => {
       const builder = new GraphBuilder();
       const name = signal('default');
       builder.addExtension(
-        'hydrator',
-        hydrator(({ id, type }) => {
-          const data = memoize(() => Math.random());
-          return { id, type, data, properties: { name: name.value } };
-        }),
-      );
-      builder.addExtension(
-        'connector',
-        connector(({ node }) => {
-          // TODO(wittjosiah): Should subscribe to parent changes.
-          const a = memoize(() => Math.random());
-          const b = memoize(() => Math.random());
-          const c = Math.random();
-          return [{ id: 'second', type: 'type', data: { a, b, c, data: node.id } }];
+        'extension',
```

Comment:
you don't need the descriminating prop now?
also pref, createExtension (and need ID?)

Reply:
these are just identifiers for the extensions, but yes they should probably just be apart of the extension def 👍